components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AuditConstants.java

```diff
@@ -37,4 +37,7 @@
     public static final String ACCOUNT_LOCKED = "Account Lock";
     public static final String ACCOUNT_UNLOCKED = "Account Unlock";
     public static final String USER_STORE_DOMAIN = "UserStoreDomain";
+    public static final String ACCOUNT_DISABLED = "Account Disable";
+    public static final String ACCOUNT_ENABLED = "Account Enable";
+    public static final String IS_MODIFIED_STATUS = "ModifiedStatus";
```

Comment:
Shall we use MODIFIED_STATUS as the variable name?

Reply:
It's named that way as it's a boolean.